TestFoundation/xdgTestHelper/main.swift

```diff
@@ -264,8 +275,11 @@ case "--nspathfor":
 #if !os(Windows)
 case "--signal-test":
     signalTest()
+
+case "--print-open-file-descriptors":
```

Comment:
This breaks the Windows build.

Reply:
@millenomi are you sure? It's in a `#if !os(Windows)` block

Reply:
I missed it right there >.<' I'll double-check the test to make sure the chain holds all the way through.

Reply:
It is! I'm not sure if we should #if !os(Windows) the test explicitly, but it works out the way the patch is.

Reply:
@weissi can you also guard the test so that it isn't invoked incorrectly?

Reply:
yes, done. thanks